File: Java/Iniciante/1010 - Cálculo Simples.java
import java.util.Scanner;

/*
 * Autor: Pierre Vieira
 * Data de submissão: 26/01/2020 13:52:03
 */
public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        String line1, line2;
        int qtde_peca1, qtde_peca2;
        float valor_peca1, valor_peca2;
        line1 = teclado.nextLine();
        line2 = teclado.nextLine();
        qtde_peca1 = Integer.parseInt(line1.split(" ")[1]);
        qtde_peca2 = Integer.parseInt(line2.split(" ")[1]);
        valor_peca1 = Float.parseFloat(line1.split(" ")[2]);
        valor_peca2 = Float.parseFloat(line2.split(" ")[2]);
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", qtde_peca1*valor_peca1 + qtde_peca2*valor_peca2);
    }
}

File: C#/Iniciante/1010 - Cálculo Simples.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 12:36:42
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int codigo_peca1, numero_pecas1, codigo_peca2, numero_pecas2;
            double valor_1, valor_2, valor_a_pagar;
            string[] linha1, linha2;
            linha1 = Console.ReadLine().Split(' ');
            linha2 = Console.ReadLine().Split(' ');
            codigo_peca1 = int.Parse(linha1[0]);
            codigo_peca2 = int.Parse(linha2[0]);
            numero_pecas1 = int.Parse(linha1[1]);
            numero_pecas2 = int.Parse(linha2[1]);
            valor_1 = double.Parse(linha1[2]);
            valor_2 = double.Parse(linha2[2]);
            valor_a_pagar = valor_1 * numero_pecas1 + valor_2 * numero_pecas2;
            Console.WriteLine($"VALOR A PAGAR: R$ {valor_a_pagar.ToString("F2", CultureInfo.InvariantCulture)}");
        }
    }
}
